recipe/meta.yaml

```
@@ -32,10 +34,16 @@ test:
 about:
   home: https://github.com/clarete/forbiddenfruit
   summary: Patch python built-in objects
-  license: LGPL-3.0-only & MIT
+  description: |
+    This project aims to help you reach heaven while writing tests, but it may lead you to hell if used on production code.
+    It basically allows you to patch built-in objects, declared in C through python.
+  license: LGPL-3.0-only OR MIT
```

```suggestion
  license: MIT
```
License is at our choice, MIT or GPLv3. https://github.com/clarete/forbiddenfruit/blob/0.1.4/README.md#license

I would choose MIT.